vulnerable_people_form/integrations/location_eligibility.py

```diff
@@ -75,3 +74,53 @@ def is_postcode_in_england(postcode):
                                    f"Postcode: {postcode} "))
 
     return is_postcode_in_england
+
+
+def get_ladcode_from_postcode(postcode):
```

Comment:
is there any test data required for this to work locally e.g see the following file https://github.com/alphagov/govuk-shielded-vulnerable-people-service/blob/master/scripts/prepopulate.sql - Does this need updating?

Reply:
not really, I dont think so... this piece of work assumes data stored in front end... for test, i created data in `tests/integrations/data/local-area-shielding-advice.csv` which we should be able to use locally as well.. unless you are looking for something more explicit...

We will though need test data same as NSPL. ONSUD or other when we move this functionality to back end. Not sure when that will be.

Reply:
I think our goal for this story was to lay the ground work as best we could to handle shielding levels in different LAs. We don't currently have "real" data so I think for the time being it makes sense not to have any. This will mean this code in this PR is effectively dead code until we get some real data but I think that's fine.